ApiDoctor.Validation/Http/HttpParser.cs

```diff
@@ -72,13 +72,17 @@ public static HttpRequest ParseHttpRequest(string requestString)
                             throw new HttpParserRequestException("Request contains an HTTP response.");
 
                         string url;
-                        var httpVersion = components[1].StartsWith("HTTP/") ? components[1] : "HTTP/1.1";
-                        if (components.Length > 3)
+                        var httpVersion = components.Last().StartsWith("HTTP/") ? components[1] : "HTTP/1.1";
```

Comment:
I think the test failure has something to do with this change. @andrueastman, can you please have a look?

Reply:
I will take a look and revert.

Reply:
@zengin this should now be addressed [here](https://github.com/microsoftgraph/apidoctor/pull/6)
Merging the PR should update this PR as well